wpilibNewCommands/src/main/native/cpp/frc2/command/Command.cpp

```diff
@@ -205,6 +207,22 @@ void Command::InitSendable(wpi::SendableBuilder& builder) {
       "runsWhenDisabled", [this] { return RunsWhenDisabled(); }, nullptr);
 }
 
+CommandPtr operator<<(CommandPtr command, auto toRun) {
+  return command.AndThen(toRun);
+}
+
+CommandPtr operator<<(Command* command, auto toRun) {
+  return command->AndThen(toRun);
+}
```

Comment:
An argument could be made that operator| should be used for this instead, because std::ranges uses operator| as a pipe operator.